sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -28,12 +31,13 @@
 import tempfile
 import time
 from datetime import datetime
-from StringIO import StringIO
+from io import BytesIO
 
 import pkg_resources
 from apitools.base.py import encoding
 from apitools.base.py import exceptions
-import six
+
+from future import standard_library
```

Comment:
Do we want to call this before we do the import of "io" above?  It looks like this only works because of an "accident" in that this module is imported after someone else already called `install_aliases()`.

Reply:
I do not believe that you can just write __from future import standard_library__.  I believe that you must write __from future.standard_library import install_aliases ; install_aliases()__ and (despite what the linters will tell you) that call must happen before you import effected modules.  If I am wrong on this the please enlighten me.

Reply:
OK, I'll remove the install_aliases caused by `io`. However regarding the order, it seems that install_aliases doesn't need to be called before the imports (copied from [here](https://github.com/apache/beam/pull/5373#discussion_r200076690)):

I tested functionality of this placement and this order works as well:
test script* (Python2):
Without install_aliases():

```>>> import sys
>>> sys.intern
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'intern'
```
With install_aliases() after import sys

```>>> import sys
>>> from future import standard_library
>>> standard_library.install_aliases()
>>> sys.intern
<built-in function intern>
```

*Based on [documented effect of aliased imports](http://python-future.org/standard_library_imports.html#aliased-imports)